webapp/components/wpt-results.html

```diff
@@ -227,13 +233,23 @@
           this.refreshDisplayedNodes();
         };
 
+        // Add a diff testRun spoof, if needed.
+        if (this.diff && this.testRuns && this.testRuns.length == 2) {
```

Comment:
If there aren't exactly two runs and the diff parameter is used, can some kind of message be shown? It could be that a URL that previously worked no longer does because a new configuration matches the labels, and that'd be a bit confusing for someone who doesn't know how the diff param works.

Reply:
Added a toast notification.

Reply:
Added a notification.